src/annotator/integrations/pdf-metadata.ts

```diff
@@ -70,9 +86,10 @@ export class PDFMetadata {
    * @param app - The `PDFViewerApplication` global from PDF.js
    */
   constructor(app: PDFViewerApplication) {
-    this._loaded = pdfViewerInitialized(app).then(() => {
+    this._loaded = pdfViewerInitialized(app).then(async () => {
```

Comment:
I'm not going to change that here, as it may have other implications, but I think that we could simply do this:

```
this._loaded = pdfViewerInitialized(app)
    .then(() => app.pdfDocument.getDownloadInfo())
    .then(() => app);
```

But we would probably have to drop support for some older versions of PDF.js

Reply:
> But we would probably have to drop support for some older versions of PDF.js

We need to clarify the supported range of PDF.js versions. I think there are some existing code comments somewhere. The main considerations would be:

1. The version of PDF.js used in Via and the current extension
2. Versions of PDF.js used in the last N Firefox releases
3. Which older versions of https://github.com/hypothesis/pdf.js-hypothes.is we want to support

Of these, (3) is the main potential source of old PDF.js versions still in use. We might need to do some metrics gathering in the client to find out how many requests are coming from old releases, and also add some kind of warning banner if the client is loaded in an old version that is unsupported.

Reply:
I didn't know https://github.com/hypothesis/pdf.js-hypothes.is existed.